BangazonAPI/BangazonAPI/Models/PaymentType.cs

```diff
@@ -11,7 +11,7 @@ public class PaymentType
         public string name { get; set; }
         public int accountNumber { get; set; }
         public int CustomerId { get; set; }
-        public Customer customerPaying { get; set; } = new Customer();
+        //public Customer customerPaying { get; set; } 
```

Comment:
Does this need to be deleted? We don't quote out lines of code.

Reply:
cut it out, good to go